deploy-cropper.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+####################################################
+# Deploy the cropper
+#
+# Requires MCD environment variables to be in scope https://changelog.makerdao.com/releases/mainnet/active/contracts.json
+#
+# Usage: ./deploy-cropper.sh
+####################################################
+
```

Comment:
Subjective so just fyi - 
Might be useful to add "set -x" at the top of such scripts as it echoes each command and you can see the actual seth commands performed with the actual addresses.

Problem here is that it will also echo the `echo` commands, so these will be ptinted twice.
But you can maybe remove them.
Anw feel free to ignore :)